### PR TITLE
Update cloud-services.md

### PR DESCRIPTION
**Pull request by nhsneal**
Added AWS references to useful services, bumped serverless higher up the list

### CODE REVIEW

File: practices/cloud-services.md [hunk 1]
```diff
@@ -13,17 +13,20 @@
 
 * Configure all infrastructure using declarative code such as Terraform and CloudFormation (see [everything as code](../patterns/everything-as-code.md)).
 * Automate monitoring and alerting (see [automate everything](../patterns/automate-everything.md)).
+* Prefer serverless platform as a service (PaaS) over infrastructure as a service (IaaS) (see [outsource bottom up](../patterns/outsource-bottom-up.md)).
+* Understand the benefits of "cattle not pets"/ephemeral architecture.
```
Comment (andyblundell) [inline practices/cloud-services.md:17]: Nice. We've got a slightly more directive "Where not serverless use ephemeral and immutable infrastructure" in the parent principles.md page - happy to swap this sentence to that?

**Review by andyblundell — CHANGES_REQUESTED** [review]

File: practices/cloud-services.md (hunk 1 above)
Comment (nhsneal) [reply practices/cloud-services.md:17]: yes sounds good Andy

**Review by ivorc — APPROVED** [review]
Good tweaks which make it clearer.

**Review by andyblundell — APPROVED** [review]

File: practices/cloud-services.md (hunk 1 above)
Comment (andyblundell) [reply practices/cloud-services.md]: 👍

**Pull request merged**